pages/app-layout/embedded-view-mode.tsx

```diff
@@ -0,0 +1,122 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
```

Comment:
How do you use this page? It does not render a route because it does not match `*.page.tsx` pattern

Reply:
I want to preview this page before approving, but I can't

Reply:
Oops, error while renaming. Updating.